### ISSUE TITLE
Issues in RGI region08 tile6 (Kebnekaise massif, Sweden)

### ISSUE THREAD

**Issue by erikmannerfelt** [GLIMS-RGI/rgi7_scripts#36]
I noticed that all glaciers near the Kebnekaise massif in Sweden were poorly georeferenced in the order of 100–200 m. It doesn't seem to be a systematic shift, but rather some projection error. Especially Storglaciären looks like it was squished by a troll! Below is the RGI6 outline of it (red) and an example of a 2017 (green) outline on top of it in 1990 (beige) that I did for a paper. One can also quite clearly see the generally eastward offset of the neighbouring glaciers, as well as the way too thick glacier body of Storglaciären.

![image](https://user-images.githubusercontent.com/33550973/148796210-860467b7-b72d-4f7a-9bc8-297128b5f98f.png)

Please comment generally on issues identified here:

I started identifying the ones that were off in the vicinity, until I realised that all of them were off! Below is just a small sample of the erroneous ones.

G018541E67940N,G018557E67946N,G018563E67935N,G018579E67937N,G018603E67938N,G018608E67947N,G018648E67934N,G018564E67915N,G018569E67903N,G018496E67910N,G018534E67896N,G018512E67971N,G018564E67972N,G018522E67944N

Thank you for your great work!

**Comment by fmaussion** [comment]
@erikmannerfelt thanks for the report! 

All outlines in Sweden are based on the GLIMS submissions 362 | 363 | 364 | 365 which seems to all come from the same batch of data (analyst, submitter): https://github.com/GLIMS-RGI/rgi7_scripts/wiki/RGI08-(Scandinavia) 

- Is it safe to say that all outlines in Sweden (not only Kebnekaise) have these issues? 

Regarding the action to take now: it depends on:
- is your inventory available for whole of Sweden?
- if it is, do we want to use it but then loose closeness to 2000?
- can someone make a new inventory for Sweden at the year 2000?
- if all fails, ignore and put this on our TODO list for RGI 7.1 ...

**Comment by fmaussion** [comment]
Outlines plotted on the GEE tool for confirmation:

![image](https://user-images.githubusercontent.com/10050469/148800142-3b31edf3-cc9f-4bc9-94de-5e2fd6f47ea0.png)

**Comment by erikmannerfelt** [comment]
@fmaussion, I checked Sarek and Sulitjelma (two other highly glaciated mountain ranges) and they also have the same issues. I would consider all Swedish outlines erroneous!

I had a quick look over the Norwegian border and those outlines look almost perfect. But as soon as it's Sweden, they are wrong! A fun example is the Salajekna glacier which is half in Norway and half in Sweden. Guess which half is good ;) 

I will contact the glaciology group in Stockholm University ASAP and ask what they think is the best way forward. My own inventory only stretches around the Kebnekaise massif, so I'm sure there's a more complete archive somewhere.

**Comment by fmaussion** [comment]
> A fun example is the Salajekna glacier which is half in Norway and half in Sweden.

Yes this is documented here: https://github.com/GLIMS-RGI/rgi7_scripts/issues/26

> I will contact the glaciology group in Stockholm University ASAP

Thanks a lot for doing this - please let us know the outcome.

**Comment by erikmannerfelt** [comment]
I've been in contact with the glaciology group in Stockholm. Basically, there is no inventory to replace the outlines on Swedish territory at the moment... Since the errors are not only in x and y, but also in their shapes, I recommend a full revision of the outlines.

I may have the time to do this during 2022, depending on some hanging decisions at SU. I'll post updates accordingly. If things go awry in this regard, I'll do my best to delegate this issue before RGI 7.1!

**Comment by fmaussion** [comment]
This problem has been partly addressed by Thomas Frank from Uppsala University.

However. visual analysis reveals issues in the southernmost glaciers (blue: RGI6, red: Thomas Frank):

![image](https://user-images.githubusercontent.com/10050469/217296642-2eee9c43-6f3b-4970-a809-c7aaf52df99b.png)

At the border to Norway, the new outline leads to overlap to Norwegian outlines:

![image](https://user-images.githubusercontent.com/10050469/217297350-fba585b5-8d3e-47b1-8fd2-bacf8ade3205.png)

**Comment by fmaussion** [comment]
As a result I will keep this issue open - I'm not sure that RGI08 in Sweden is in the best possible state
